scipy/stats/tests/test_distributions.py

```diff
@@ -6054,6 +6054,17 @@ def test_support_broadcasting_gh13294_regression():
     assert b2.shape == ex_b2.shape
 
 
+def test_stats_broadcasting_gh14953_regression():
+    # test case in gh14953
+    loc = [0., 0.]
+    scale = [[1.], [2.]]
+    assert_equal(stats.norm.var(loc, scale), [[1., 1.], [4., 4.]])
```

Comment:
Not a big deal, but I would add a different dimension to check it's not just a transpose issue.
Without your change it would return
```python
array([[1., 4.],
       [9., 1.],
       [4., 9.]])
```

```suggestion
    scale = [[1.], [2.], [3.]]
    assert_equal(stats.norm.var(loc, scale), [[1., 1.], [4., 4.], [9., 9.]])
```